src/Card/Card.js

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { FlexBox } from "../FlexBox/FlexBox";
+import { Heading } from "../Heading/Heading";
+import { cardAlignOptions } from "./Card.levers";
+
+/**
+ * Card Component
+ *
+ * HTML card element
+ */
+
+export const Card = ({ className, children, isMinimal, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+
+  let wrapperClasses =
+    "su-bg-white su-text-black su-border su-border-solid su-border-black-10 su-shadow";
+
+  if (isMinimal) {
+    wrapperClasses = "";
+  }
+
+  return (
+    <article
```

Comment:
It probably makes sense to make the outer wrapper element configurable here. eg: [div, span, li, article, etc]